src/component/Tooltip.tsx

```diff
@@ -38,7 +38,12 @@ function renderContent<TValue extends ValueType, TName extends NameType>(
 }
 
 export type TooltipProps<TValue extends ValueType, TName extends NameType> = ToltipContentProps<TValue, TName> & {
-  active?: boolean;
+  /**
+   * If true, then Tooltip is always displayed, once an activeIndex is set by mouse over, or programmatically.
+   * If false, then Tooltip is never displayed.
+   * If active is undefined, Recharts will control when the Tooltip displays. This includes mouse and keyboard controls.
+   */
```

Comment:
This nicely shows up in the Storybook ArgsTable. As a reminder: We would like to have such comments above every single prop that is user facing, because then it is documented in the code and in storybook identically.